### DISCUSSION TITLE
Removing a Duplicate Collection Event

### DISCUSSION THREAD

**Discussion by TransientWolf** [mampfes/hacs_waste_collection_schedule#1564] [Q&A]
My provider correctly schedules my Garbage and Recycling Bin dates, but because they have a service for either 1 or 2 Garden bins, and I have 2 garden bins, they create two events on the same day.  So when showing the upcoming collections it shows for example "Green Bin in 5 days time and Brown Bin, Brown Bin in 12 days time".  Any ideas on how do I get it to just say Brown Bin once by for example ignoring duplicate events?

**Comment by 5ila5** [comment]
If both Brown Bin events have a slightly different name you can use the customize show false attribute.
If both Brown Bin events have exactly the same you may be able to change the duplicates using some advanced templateing (not tested, not sure if it works at all)
If both does not work you can give me the name of the source and I can add a duplicate filtering directly while fetching the dates, as it does not sound like there is any use to have the exactly same event twice?

**Reply by TransientWolf** [reply]
Hi. Thanks for answer.  They both have the same data identically and come through as "Empty Bin 240L Brown" and it is Peterborough Council.  I use an alias to change to "Brown Bin" but of course both change to Brown Bin and show in the calendar.
